unfoldingWord/translationCore
# Fix For \q tags and \f tags and Address Infinite Render Loop

#### This pull request addresses:
This PR fixes an issue with the \q tags filter function removing a needed space, and it also fixes a bug which makes the Scripture Pane endlessly re-render.


#### How to test this pull request:
run ```npm update usfm-js```
Test by opening tW in Ephs and going to the ```Christ``` check, ensure that the verse reads ```5:14 For anything that becomes visible is light. Therefore it says, "Awake, you sleeper, and arise from the dead, and Christ will shine on you."```


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3287)
